src/scenes/AcceleratorRouter/ParticipantProjects/EditView/index.tsx

```diff
@@ -14,48 +16,67 @@ import PageForm from 'src/components/common/PageForm';
 import useNavigateTo from 'src/hooks/useNavigateTo';
 import { requestUpdateParticipantProject } from 'src/redux/features/participantProjects/participantProjectsAsyncThunks';
 import { selectParticipantProjectUpdateRequest } from 'src/redux/features/participantProjects/participantProjectsSelectors';
+import { requestProjectUpdate } from 'src/redux/features/projects/projectsAsyncThunks';
+import { selectProjectRequest } from 'src/redux/features/projects/projectsSelectors';
 import { useAppDispatch, useAppSelector } from 'src/redux/store';
 import strings from 'src/strings';
 import useForm from 'src/utils/useForm';
 import useSnackbar from 'src/utils/useSnackbar';
 
+import { useScoringData } from '../../Scoring/ScoringContext';
+import { useVotingData } from '../../Voting/VotingContext';
 import { useParticipantProjectData } from '../ParticipantProjectContext';
 import EditNameConfirm from './EditNameConfirm';
 
 const EditView = () => {
   const dispatch = useAppDispatch();
   const theme = useTheme();
   const snackbar = useSnackbar();
-  const { crumbs, projectId, project } = useParticipantProjectData();
+  const { crumbs, projectId, participantProject, project, organization, reload } = useParticipantProjectData();
+  const { phase1Scores } = useScoringData();
```

Comment:
Do we anticipate needing to show scoring for projects outside of phase1?

Reply:
I believe each phase except 0 will have scores.